src/components/Popup.js

```diff
@@ -1,11 +1,20 @@
 export default class Popup {
   constructor({ popupSelector }) {
     this._popupElement = document.querySelector(popupSelector);
+    this._submitButton = this._popupElement.querySelector(
```

Comment:
Needs correction: the Popup should not know anything about the submit buttons inside the specific popups